### PR TITLE
Remove deprecated config/plugin from config-reloader templates.

### PR DESCRIPTION
**Pull request by raja-gola**
Signed-off-by: Raja Sekhar <rajasekhar@vmware.com>

This addresses https://github.com/vmware/kube-fluentd-operator/issues/108
Please have a look.

### CODE REVIEW

**Comment by shukla2009** [comment]
any time line for merging ?

**Comment by raja-gola** [comment]
@carpenterm @OrlinVasilev

**Comment by carpenterm** [comment]
@raja-gola thanks for the PR, changing the log format output is not something to be taken lightly. We want to do our due diligence on this and understand the impact for our teams already using kube-fluentd-operator before merging. Once we are confident of a path forward we will merge. @shukla2009 I'll get back to you with an ETA once we have prioritized testing this internally and have a better understanding of it's impact, but I am hoping we can do that in the next week or two.

File: config-reloader/templates/kubernetes-postprocess.conf [hunk 1]
```diff
@@ -49,14 +40,3 @@
...
 </filter>
 
-# Parse logs in the kube-system namespace using the kubernetes formatter.
-<filter kube.kube-system.**>
-  @type parser
-  reserve_data true
-  key_name log
-  emit_invalid_record_to_error false
-  <parse>
-    @type kubernetes
```
Comment (dimalbaby) [inline config-reloader/templates/kubernetes-postprocess.conf:59]: @raja-gola I have looked at the change. I think it breaks number additional fields populated as part of the custom kubernetes parse plugin in kube-system ns. The parser removes the 'log' key and add some extra fields which will then be attached to the new key 'message'. I am planning to make a new release by end of this week. Would you mind trying again on the latest release ?

Comment (dimalbaby) [reply config-reloader/templates/kubernetes-postprocess.conf:59]: @shukla2009 Can you provide more details about the issue? The details would be really helpful for finding a solution

**Comment by Cryptophobia** [comment]
Partially addressed in #115 and removes the `merge_json_log` , `preserve_json_log` following just [fluent-kubernetes-metadata_plugin](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter) configuration.

> NOTE: As of the release 2.1.x of this plugin, it no longer supports parsing the source message into JSON and attaching it to the payload. The following configuration options are removed:
> 
>     merge_json_log
>     preserve_json_log
> 
> One way of preserving JSON logs can be through the parser plugin

**Pull request closed without merging**